### PR TITLE
Apply google-java-format to spotlessApply step on gradle build.

### LINKED ISSUE #163: Spotless Not Running google-java-format
### Description
The README mentions

> We use Google's Java coding conventions for the project

however, our spotless gradle-plugin configuration is not running google-java-format.

### Acceptance Criteria
* `./gradlew spotlessApply` should execute google-java-format and format project code to be compliant with the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html).

### Steps to Reproduce (Bug)
1. Run `./gradlew spotlessApply` with a code snippet which would be caught by google-java-format.
2. Note that no action is taken.

**Expected behavior:** `google-java-format` is executed and code is formatted to comply with the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html).

**Actual behavior:** Only 5 spotless checks are run:

1. removeUnusedImports
2. importOrder
3. trimTrailingWhitespace
4. endWithNewline
5. licenseHeaderFile

**Frequency:** 100%

### PR DESCRIPTION
**Pull request by schroedingerscode**
## PR Description
Automatically applies [google-java-format](https://github.com/google/google-java-format) to spotlessApply step as part of gradle build. The first commit is the only change that was made manually (`/build.gradle`). The second commit were changes made automatically as part of the spotlessApply step.

## Fixed Issue(s)
Fixes #163

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -74,6 +74,7 @@ allprojects {
       trimTrailingWhitespace()
       endWithNewline()
       licenseHeaderFile "${rootDir}/gradle/spotless.java.license"
+      googleJavaFormat().aosp()
```
Comment (jrhea) [inline build.gradle:77]: We might want to recommend that everyone with one PRs add this to their build.gradle file and run the spotless step to minimize the conflicts.

**Review by jrhea — COMMENTED** [review]
I think we need @akhila-raju and @dangerousfood to take a look to make sure that this doesn't make it a nightmare to merge their PRs

File: build.gradle (hunk 1 above)
Comment (schroedingerscode) [reply build.gradle:77]: Yeah I agree. This is an annoying change, but luckily we only have to do it once.

Comment (akhila-raju) [reply build.gradle:77]: Is it ok if we put merging this on hold for a bit?

Comment (schroedingerscode) [reply build.gradle:77]: Yep. In fact, jenkins/gradle is being a bit odd, so I'm gonna close this PR and I'll recreate it at another time.

**Comment by schroedingerscode** [comment]
Closing for time being, will reopen at another time when there is less in flight.

**Pull request closed without merging**